packages/@sanity/components/src/toggles/styles/Checkbox.css

```diff
@@ -17,7 +17,7 @@
   z-index: 1;
   display: inline-block;
   box-sizing: border-box;
-  height: var(--checkbox-label-height);
+  /* height: var(--checkbox-label-height); */
```

Comment:
Can this be removed? Or if it needs to stay - add a comment about why?

Reply:
It shoudl go